spec/generators/post_deployment_migration_generator_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'rails/generators/testing/behaviour'
+require 'rails/generators/testing/assertions'
+
+require 'generators/post_deployment_migration/post_deployment_migration_generator'
+
+describe PostDeploymentMigrationGenerator, type: :generator do
+  include Rails::Generators::Testing::Behaviour
+  include Rails::Generators::Testing::Assertions
+  include FileUtils
+
+  tests described_class
+  destination File.expand_path('../../tmp', __dir__)
+  before { prepare_destination }
+  after { rm_rf(destination_root) }
+
+  describe 'the migration' do
+    it 'generates expected file' do
+      run_generator %w(Changes)
+
+      assert_migration('db/post_migrate/changes.rb', /disable_ddl/)
+      assert_migration('db/post_migrate/changes.rb', /change/)
```

Comment:
Shouldn't these paths include a timestamp prefix?

Reply:
The generator continues to produce output files with timestamp prefixes -- but in the spec (since they vary every generation) there's an underlying file path helper method used by `assert_migration` so you don't need to know what the full path is. I think will regex match against any numeric preceded string (which matches both the old style integer numbering and the newer timestamps).

Reply:
Oh, I see! Thanks for the information.